### PR TITLE
Allow localhost testing without a TLS cert.

### PR DESCRIPTION
**Pull request by twifkak**
Enable a `CertURLBase = "data:"` option that allows http://localhost SXGs to
display in the browser.

I tested the error plumbing by injecting a fake error in ExchangeMetaFactory.Get().

/cc @khempenius

### CODE REVIEW

File: server/exchange.go [hunk 1]
```diff
@@ -72,20 +75,38 @@ func NewExchangeMetaFactory(c ExchangeConfig) *ExchangeMetaFactory {
...
 	chain := e.CertManager.GetAugmentedChain()
 
-	// Use path.Join so the last path element in e.CertURLBase is kept
-	// whether or not it has the trailing slash.
-	urlPath := path.Join(e.CertURLBase.Path, chain.Digest)
+	var certURL *url.URL
+	if e.CertURLBase.Scheme == "data" {
+		var cbor bytes.Buffer
+		if err := chain.WriteCBOR(&cbor); err != nil {
+			return nil, fmt.Errorf("error creating data: cert-url: %w", err)
```
Comment (banaag) [inline server/exchange.go:85]: webpackager uses xerrors.Errorf

File: packager.go [hunk 2]
```diff
@@ -62,7 +63,10 @@ func (pkg *Packager) Run(url *url.URL, sxgDate time.Time) error {
 // RunForRequest uses req directly: RequestTweaker mutates req; FetchClient
 // sends req to retrieve the HTTP response.
 func (pkg *Packager) RunForRequest(req *http.Request, sxgDate time.Time) error {
-	runner := newTaskRunner(pkg, sxgDate)
+	runner, err := newTaskRunner(pkg, sxgDate)
+	if err != nil {
+		return fmt.Errorf("packaging: %w", err)
```
Comment (banaag) [inline packager.go:68]: use xerrors.Errorf

File: server/exchange.go (hunk 1 above)
Comment (banaag) [inline server/exchange.go:85]: xerrors

Comment (twifkak) [reply server/exchange.go]: Done.

File: packager.go (hunk 2 above)
Comment (twifkak) [reply packager.go]: Done.

**Review by banaag — APPROVED** [review]

**Pull request merged**